tests/test_isdtype.py

```diff
@@ -61,12 +59,12 @@ def isdtype_(dtype_, kind):
         res = dtype_categories[kind](dtype_)
     else:
         res = dtype_ == kind
-    assert type(res) is bool
+    assert isinstance(res, bool)
```

Comment:
This should not be changed. This is a test. We want the type to be *exactly* `bool`.

Reply:
I see. This is highlighted as an error in ruff, but in this case it is the desired behavior. I will add an ignore comment instead.

Reply:
Done.